app/api/attendees.py

```diff
@@ -170,6 +170,21 @@ def before_update_object(self, obj, data, kwargs):
 #         if not has_access('is_registrar', event_id=obj.event_id):
 #         raise ForbiddenException({'source': 'User'}, 'You are not authorized to access this.')
 
+        if 'ticket' in data:
+            user = safe_query(self, User, 'id', current_identity.id, 'user_id')
+            ticket = db.session.query(Ticket).filter_by(
+                id=int(data['ticket']), deleted_at=None
+                ).first()
+            if ticket is None:
+                raise UnprocessableEntity(
+                    {'pointer': '/data/relationships/ticket'}, "Invalid Ticket"
+                )
+            if not user.is_verified and ticket.price == 0:
```

Comment:
@shreyanshdwivedi just want to ask, what will happen in the case where a guest user decides to buy a ticket. AFAIK, they will be stopped from buying any ticket if they are in combination with a free order too?

Reply:
Yes @uds5501. When an order is made which have multiple tickets, at first a PATCH request is sent to add the info of the attendees and then order object is created.
But we are raising error at the addition of attendees itself.
I've tested a combination of paid and free tickets and error was raised. When I checked orders, no order object was created.
However please test it on local and confirm if this implementation is working for u or not.